downstream/modules/hub/proc-create-namespace.adoc

```diff
@@ -11,10 +11,10 @@ When creating a namespace, you can assign a group in {HubName} as owners of that
 
 .Procedure
 //[ddacosta] For 2.5 this will be Log in to Ansible Automation Platform and select Automation Content. Automation hub opens in a new tab. From the navigation ...
-. Log in to your {PrivateHubName}.
+. Log in to your {PlatformNameShort}.
 . From the navigation panel, select {MenuACNamespaces}.
```

Comment:
This procedure is different now:
. Click btn:[Create namespace] and enter a *Name*. 
. Optionally enter a *Desription* and other details in the available fields.
. Click btn:[Create namespace].
. Select the *Team Access* tab to assign team roles.
 . Click btn[Add roles].
. Select the team that will receive new roles and click btn:[Next].
. Select the roles to be applied to the selected team and click btn:[Next].
. Review the settings and click btn:[Finish].